src/ServiceProvider.php

```diff
@@ -54,6 +55,19 @@ public function logger()
         EventStore::logger();
     }
 
+    /**
+     * Handle event events.
+     *
+     * @return void
+     */
+    public function events()
```

Comment:
Would prefer to name this per `setX`

Reply:
The only reason I didn't change it to be like that is because I wanted to follow the `Laravel Way`

Please see `https://github.com/spatie/freek.dev/blob/master/app/Providers/NovaServiceProvider.php`

Reply:
Mhm - need to get used to this. Good that they still do `[verb]X` on protected methods